interface/src/app/gene/gene-network.tpl.html

```diff
@@ -0,0 +1,10 @@
+<div>
+  <h1 class="page-header">
+    Gene Network
+  </h1>
+  <div class="well">
+    <geneset-search-form ng-if="userObj"></geneset-search-form>
```

Comment:
Ok, I will leave it as it is for the purposes of this PR, and if we decide that it would be more user friendly to change it, I think it will be a pretty easy change.